manifest.json

```diff
@@ -257,6 +298,32 @@
         }
       ]
     },
+    {
+      "label": "header_page_group_label",
+      "variables": [
+        {
+          "identifier": "show_user_info",
+          "type": "checkbox",
+          "description": "user_info_description",
+          "label": "user_info_label",
+          "value": false
+        },
+        {
+          "identifier": "mit_link",
+          "type": "text",
+          "description": "mit_link_description",
+          "label": "mit_link_label",
+          "value": "https://mit.edu/"
+        },
```

Comment:
Are we using this now?

Reply:
This setting is not used and has been removed.